### PR TITLE
fix: pipeline

### PR DESCRIPTION
**Pull request by ntn-x2**

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -287,34 +287,5 @@ module.exports = {
         },
       },
     ],
-    [
```
Comment (Dudleyneedham) [inline docusaurus.config.js:290]: What was this?

Comment (ntn-x2) [reply docusaurus.config.js:290]: Yeah it was pulling in some READMEs from the chain repo and updating some links, but the final document looks exactly the same with and without it, so I just removed it.

Comment (Dudleyneedham) [reply docusaurus.config.js:290]: I should ask what was it intending to do

Comment (ntn-x2) [reply docusaurus.config.js:290]: Then my previous comment still answers your question, I think 🤔

**Review by Dudleyneedham — APPROVED** [review]
LGTM

**Pull request merged**